Task_1/Task_1/src/components/Input_form/form.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+
+const Form = (props) => {
+  const [enterTask, setEnterTask] = useState("");
+  const [enterDate, setEnterDate] = useState("");
+
+  function TaskHandler(event) {
+    setEnterTask(event.target.value);
+  }
+  function DateHandler(event) {
+    let date = event.target.value.toString();
+    setEnterDate(date);
+  }
+
+  function add_task() {
+    if(enterTask=="" || enterDate == ""){
```

Comment:
it can be directly written as (!enterTask || !enterDate)

Reply:
Implemented